src/charts/bar.js

```diff
@@ -164,21 +186,25 @@ define(function(require) {
          * @param {BarChartData} _data The data to attach and generate the chart
          */
         function exports(_selection) {
+            if (locale) {
+                localeFormatter = setDefaultLocale(locale);
+            }
+
```

Comment:
FYI These lines were added, but not tested.  There are more like this, but I won't call them out.

Reply:
there's a ton of them.  I know we need to check if things actually exist on the chart, and even on print mode, there's some tests that could be run too.